autogen/agentchat/contrib/web_surfer.py

```diff
@@ -9,13 +9,16 @@
 from typing_extensions import Annotated
 
 from ... import Agent, AssistantAgent, ConversableAgent, GroupChat, GroupChatManager, OpenAIWrapper, UserProxyAgent
-from ...browser_utils import SimpleTextBrowser
+from ...browser_utils.bing_browser import BingTextBrowser
+from ...browser_utils.google_broswer import GoogleTextBrowser
 from ...code_utils import content_str
 from ...oai.openai_utils import filter_config
 from ...token_count_utils import count_token, get_max_token_limit
 
 logger = logging.getLogger(__name__)
 
+BROWSERS = {"google": GoogleTextBrowser, "bing": BingTextBrowser}
```

Comment:
here enums too, or at least some configuration so that users can register new text browsers if they implement new ones (like wikimedia servers for example) maybe a better idea is to pass the textbroser instance to the agent factory itself